Assets/Scripts/Bullet/Bullet.cs

```diff
@@ -17,9 +17,21 @@ private void Awake()
 
     private void Start()
     {
-        Destroy(gameObject, lifetime);
-        Vector2 direction = transform.localScale.x > 0 ? Vector2.right : Vector2.left;
+    
+    }
+
+    public void Initialize(Vector2 shootDirection)
```

Comment:
Can we revert this to my changes on main? Instead of initialize, we just set the flip in the Instantiate call. It simplifies the looper logic.

Reply:
@faizahsayyid addressed here https://github.com/faizahsayyid/gmtk-2025/pull/8/commits/516cf71d43119e5c2fb55a4a318f1ed17c2c1504

Reply:
Try that and see if you get the same issue

Reply:
@faizahsayyid the issue is still there

https://github.com/user-attachments/assets/afe55728-a225-4ed9-9968-3d025b62f2e5